pkg/openscap/openscap_test.go

```diff
@@ -0,0 +1,70 @@
+package openscap
+
+import (
+	"fmt"
+	//docker "github.com/fsouza/go-dockerclient"
+	"testing"
+)
+
+type testScanner struct {
+	defaultScanner
+
+	getRHELDistOriginal bool
```

Comment:
Maybe consider making these functions then you can mock in any behavior you want later and not have to worry about the flags and then test scanner doesn't even need to know about the default scanner impl.

Something like:

```go
type testScanner struct {
   oscapChroot func (oscapArgs ...string) ([]byte, error) 
}

func (s *testScanner) oscapChroot(oscapArgs ...string) ([]byte, error) {
 	return s.oscapChroot(oscapArgs)
}

func MyTest() {
   errorScanner := &testScanner {
       oscapChroot: func(oscapArgs ...string) ([]byte, error) { return nil, fmt.Errorf("foo!")}
   }

   realScanner := nil
   myDelegatingScanner := &testScanner {
       oscapChroot: func(oscapArgs ...string) ([]byte, error) { return realScanner.oscapChroot(oscapArgs) }
   }
}
```